gradle/libs.versions.toml

```diff
@@ -1,28 +1,64 @@
 [versions]
 kotlin = "1.7.20"
-kotest = "5.5.1"
 kotlinx-serialization = "1.4.1"
 
+kotest = "5.5.1"
+kotest-propertyArbsExtension = "2.1.2"
+kotest-springExtension = "1.1.1"
+kotest-wiremockExtension = "1.0.3"
+
+gradlePlugin-testlogger = "3.2.0"
+
 
 [libraries]
 
-# KotlinX Serialization
-kotlinx-serialization-core = { module = "org.jetbrains.kotlinx:kotlinx-serialization-core", version.ref = "kotlinx-serialization" }
-kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json", version.ref = "kotlinx-serialization" }
+## Kotlin stdlib ##
+kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
```

Comment:
Which artifacts are included in this one?

Reply:
You can check in the POM https://search.maven.org/artifact/org.jetbrains.kotlin/kotlin-bom/1.7.20/pom

I think it's basically everything except for Kotlinx projects, and the Gradle plugins.